src/runtime-rs/crates/hypervisor/src/qemu/cmdline_generator.rs

```diff
@@ -821,6 +821,29 @@ impl ToQemuParams for NumaNode {
     }
 }
 
+#[derive(Debug)]
+struct MemoryBackendMachine {
```

Comment:
Hm, we already have a `-machine` generator (see `struct Machine` elsewhere in this file), why don't you just extend that instead of adding another one?

Reply:
yep, I've incorporated `struct MemoryBackendMachine` into `struct Machine`. Thanks!